pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -478,32 +478,6 @@ type SeedConfig struct {
 	gardencorev1beta1.SeedTemplate `json:",inline"`
 }
 
-// FluentBit contains configuration for Fluent Bit.
-type FluentBit struct {
-	// ServiceSection defines [SERVICE] configuration for the fluent-bit.
-	// If it is nil, fluent-bit uses default service configuration.
-	// +optional
-	ServiceSection *string `json:"service,omitempty" yaml:"service,omitempty"`
-	// InputSection defines [INPUT] configuration for the fluent-bit.
-	// If it is nil, fluent-bit uses default input configuration.
-	// +optional
-	InputSection *string `json:"input,omitempty" yaml:"input,omitempty"`
-	// OutputSection defines [OUTPUT] configuration for the fluent-bit.
-	// If it is nil, fluent-bit uses default output configuration.
-	// +optional
-	OutputSection *string `json:"output,omitempty" yaml:"output,omitempty"`
-	// NetworkPolicy defines settings for the fluent-bit NetworkPolicy.
-	// +optional
-	NetworkPolicy *FluentBitNetworkPolicy `json:"networkPolicy,omitempty" yaml:"networkPolicy,omitempty"`
-}
```

Comment:
How can operators configure these settings now with this PR?

Reply:
They can deploy the CustomResources directly in the cluster the FB operator will plug them into the fluent-bit.

But this is not a common use case. The logging stack is cloud agnostic and should have the same configuration on all providers. This workaround above was created in case we want to debug something on production landscapes and we do not want to do this with PR and wait for regular release... That's why I decided to remove this complexity from the code

Reply:
> This workaround above was created in case we want to debug something on production landscapes and we do not want to do this with PR and wait for regular release...

Interesting, I didn't know. Otherwise, we should probably never have merged such technical debt in the first place.

Anyways, it's an incompatible change now. We can do this easily since the configs are alpha-versioned, but we probably should provide some docs with information about how we expect this to be configured for people that rely on it. What's the approach to get such configurations (custom resources, network policys, ...) onto the landscapes (especially in the light of the initial motivation for these APIs)?